lib/api_account.go

```diff
@@ -26,31 +28,169 @@ func GetAccountHandler(s *sebakstorage.LevelDBBackend) http.HandlerFunc {
 		}
 
 		var err error
-
 		var ba *BlockAccount
-		if ba, err = GetBlockAccount(storage, address); err != nil {
-			http.Error(w, "Error reading request body", http.StatusInternalServerError)
-			return
-		}
-
-		var s []byte
-		if s, err = ba.Serialize(); err != nil {
-			http.Error(w, "Error reading request body", http.StatusInternalServerError)
-			return
-		}
 
 		switch r.Header.Get("Accept") {
 		case "text/event-stream":
-			event := fmt.Sprintf("saved-%s", address)
+
+			var readyChan = make(chan struct{})
```

Comment:
To handle previous data which already stored in the block,
Getting the information trigger "iterate" event